libopenage/renderer/opengl/render_pass.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017-2019 the openage authors. See copying.md for legal info.
+
+#include "render_pass.h"
+
+namespace openage {
+namespace renderer {
+namespace opengl {
```

Comment:
c++17 allows nested namespaces like this:
```
namespace openage::renderer::opengl {...}
```

we introduce this over the whole codebase in #1171, but as this is a new file you should do it right away

Reply:
this nested namespace stuff i would do in another PR with format fixing stuff. 1171 is deeper in the codebase and some namespace in the files we already changed things in. so while we were there I also updated the namespaces. but true, we should do it now right away <3